test-jmeter/helm-chart/jmeter-performance/Chart.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: v2
+name: jmeter-performance
+description: JMeter Performance Testing Framework for DotCMS Analytics APIs
+type: application
+version: 1.0.0
+appVersion: "5.6.2"
+keywords:
+  - jmeter
+  - performance
+  - testing
+  - dotcms
+  - analytics
+home: https://github.com/your-org/jmeter-performance
+sources:
+  - https://github.com/your-org/jmeter-performance
+maintainers:
+  - name: DotCMS Team
+    email: contact@example.com
```

i know these get overwritten, but should we just give reasonable values anyway?

```suggestion
home: https://github.com/dotcms/core/test-jmeter
sources:
  - https://github.com/dotcms/core/test-jmeter
maintainers:
  - name: DotCMS SaaS & Platform Team
    email: teamplatform@dotcms.com
```